feature_engine/creation/custom_functions.py

```diff
@@ -0,0 +1,10 @@
+"""
+CustomFunctions is a wrapper, which allows MathFunctions to detect if a
+custom function should be processed via pandas.agg() or numpy.apply_over_axes()
+"""
+
+
+class CustomFunctions:
```

Comment:
this file shouldnt be here, will generate conflicts when we merge the other PR

Reply:
file removed from this branch